apis/logging/v1/filter_api_audit_types.go

```diff
@@ -0,0 +1,69 @@
+package v1
+
+import auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
+
+// KubeAPIAudit filter Kube API server audit logs, as described in [Kubernetes Auditing].
+//
+// # Policy Filtering
+//
+// Policy event rules are the same format as the [Kube Audit Policy] with some minor extensions.
+// The extensions are described here, see the [Kube Audit Policy] for the standard rule behavior.
+// Rules are checked in order, checking stops at the first matching rule.
+//
+// The `level` of the rule determines how the event is treated:
+//   - None: the event is dropped.
+//   - Metadata: Audit metadata only, request and response objects are removed.
+//   - Request: Audit metadata and request are retained, response is removed.
+//   - RequestResponse: All data is retained, including the full response. This can lead to very large events.
+//     For example the command `oc list -A pods` results in a response containing the YAML description of every pod in the cluster.
```

Comment:
audit response (or this reads like the response of the oc command, which is what's intended)

Reply:
Changed text to be clearer. The response body we are talking about actually is the literal output of the oc command if you use the '-o json' option, it gets included inside the audit event at the RequestResponse level.
The API servers don't really "log" in the normal sense, they dump a complete record of every request they receive and the response they send (by default.) Hence need for filtering...